src/scss/components/_progress-indicator.scss

```
@@ -84,8 +84,14 @@
 
 .#{prefix('progress-indicator-item')} {
   display: flex;
+  padding: 10px;
```

Maybe good to use the `@include margin` that was used in the list css?

Maybe like this?
```suggestion
@include margin($grid-margin-1, $properties: padding);
```